created new folder to TIF downloads

Jason-Perez27 opened the pull request:
new scripts

Jason-Perez27 committed:
created new folder to TIF downloads

Jason-Perez27 committed:
Updated 2_data_request and created 3_labeling

anna-boser commented on code/2_data_request/3_download_timeseries.py:
start with the stratified pivots

Jason-Perez27 committed:
Updated 2_data_request and 3_labeling scripts

Jason-Perez27 committed:
-Update ReadME
-Update Data Download Pipeline to exclude Landsat 9
-Added scripts for cleaner execution of Google Earth Engine Scripts
-Improve Functionality of tif_visualizer.py
-Added new scripts http_server.py and tasks_json.py for cleaner LabelStudio Export
-Initial Code for Collection 2 pipeline (still in process of debugging and modifying for collection 2)

Jason-Perez27 committed:
Updated pull request with updated scripts in 3_labeling and 4_data_pipeline
